--- a/build.zig
+++ b/build.zig
@@ -68,3 +68,3 @@
     const enable_cplusplus = b.option(bool, "enable_cplusplus", "C++ support") orelse false;
-    const linkage = b.option(std.builtin.LinkMode, "linkage", "Build shared libraries (otherwise static ones)") orelse .dynamic;
+    const linkage = b.option(std.lang.LinkMode, "linkage", "Build shared libraries (otherwise static ones)") orelse .dynamic;
     const build_cord = b.option(bool, "build_cord", "Build cord library") orelse true;
@@ -258,3 +258,3 @@
     }
-    if (optimize != .Debug) {
+    if (optimize != .debug) {
         flags.append(b.allocator, "-D NDEBUG") catch unreachable;
